src/openvic-simulation/economy/Building.hpp

```diff
@@ -18,7 +18,16 @@
 namespace OpenVic {
 
 	struct BuildingManager;
-	struct BuildingType;
+
+	struct BuildingType : HasIdentifier {
```

Comment:
Building.*pp should probably be separated into BuildingType & BuildingInstance files.

Reply:
I guess yeah, BuildingManager doesn't rely on BuildingInstance, this PR didn't introduce that arrangement but since I'm already rearranging Building.hpp won't hurt to force push another change